PyOpenWorm/dataObject.py

```diff
@@ -130,10 +130,10 @@ def _find_base_namespace(cls, dct, bases):
     def contextualize_class_augment(self, context):
         if context is None:
             return self
+        dct = dict(rdf_namespace=self.rdf_namespace, rdf_type=self.rdf_type, class_context=context.identifier)
```

Comment:
I just tested this actually, and the re-implementation here is unneeded. This was one of the earlier pieces of code I wrote when adding contexts -- possibly the need for this was removed during implementation.